Source/WTF/wtf/FixedVector.h

```diff
@@ -29,10 +29,11 @@
 
 namespace WTF {
 
-template<typename T>
+template<typename T, typename Malloc>
 class FixedVector {
 public:
-    using Storage = EmbeddedFixedVector<T>;
+    using Storage = EmbeddedFixedVector<T, Malloc>;
+    using Base = FixedVector<T, Malloc>;
```

Comment:
Base means base class.  This is not the base class.  Calling it `Base` would be misleading.  Maybe call it `Self`?  There's no precedence for this.  In WebKit code, we typically spell it out, though that can get hairy when the template arguments are long.  Calling it `Self` or `This` might be ok.

Reply:
I picked this up from VectorBuffer but I see that it's indeed using it for the proper base-class. Will change